src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +18,34 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static final double ROBOT_TIMEOUT = 0.02; // [s]
```

Comment:
So how would I call it, if I will use one of those words, it would make it harder to understand the conjunction of the meaning and the units.

Reply:
Hmm, that's correct. 
You can use "timestep", "time interval", or "time period". But, it might be best to use "loop period" – that's the term used in the WPILib Docs (see [here](https://docs.wpilib.org/en/stable/docs/software/convenience-features/scheduling-functions.html?highlight=notifier#scheduling-functions-at-custom-frequencies) for example).